libraries/botbuilder-dialogs-adaptive/src/jsonExtensions.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+
+import { DialogStateManager } from 'botbuilder-dialogs';
+import { ExpressionParser, ValueExpression } from 'adaptive-expressions';
+
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+export class JsonExtensions {
+    /**
+     * Replaces the binding paths in a JSON value with the evaluated results recursively. Returns the final JSON value.
+     * @param dc A scope for looking up variables.
+     * @param unit A JSON value which may have some binding paths.
+     * @returns Deep data binding result.
+     */
+    public static replaceJsonRecursively(state: DialogStateManager, unit: object): any {
```

Comment:
Please refactor to follow this pattern: https://github.com/microsoft/botbuilder-js/blob/891ddc887e631b1f7534bb82ef3b7d5a6423a756/libraries/botbuilder-dialogs-adaptive/src/telemetryExtensions.ts#L24